bin/hveto

```diff
@@ -20,37 +20,31 @@
 """Run the HierarchichalVeto (hveto) algorithm over some data
 """
 
-from __future__ import (division, print_function)
-
 import time
 jobstart = time.time()
 
-import argparse
 import os
+import sys
+import json
 import random
 import warnings
-import multiprocessing
-import json
+import argparse
 import datetime
 import subprocess
-import sys
+import configparser
+import multiprocessing
+from pathlib import Path
 from socket import getfqdn
 from getpass import getuser
 from distutils.spawn import find_executable
-from pathlib import Path
-
-try:
-    import configparser
-except ImportError:  # python 2.x
-    import ConfigParser as configparser
 
 import numpy
 
 from matplotlib import use
 use('agg')
 
-from gwpy.io.cache import read_cache
 from gwpy.time import to_gps
+from gwpy.io.cache import read_cache
```

Comment:
Why was this moved?

Reply:
@duncanmmacleod, mostly it's because I'm obsessive about the flow of text width in import statements -- I like to go from narrow to wide, for cleanliness of presentation

Reply:
I prefer to go alphabetically, but I don't really care, just was curious since this isn't strictly python3-related.